src/TableFunctions/TableFunctionMongoDB.cpp

```diff
@@ -91,7 +91,7 @@ void TableFunctionMongoDB::parseArguments(const ASTPtr & ast_function, ContextPt
             const auto & [arg_name, arg_value] = getKeyValueMongoDBArgument(ast_func);
             if (arg_name == "structure")
                 structure = checkAndGetLiteralArgument<String>(arg_value, arg_name);
-            else if (arg_name == "options" || arg_name == "oid_columns")
+            else 
                 main_arguments.push_back(arg_value);
```

Comment:
Is it working?
We keep in args only the value of the arg. How does the named collection manage to override values without a key?